readme.md

```diff
@@ -2,56 +2,56 @@
 
 Merges the enumerable properties of two or more objects deeply.
 
-> UMD bundle is 723B minified+gzipped
+> UMD bundle is 792B minified+gzipped
 
 ## Getting Started
 
 ### Example Usage
+
 <!--js
-const merge = require('./')
+const { default: deepmerge, deepmergeAll } = require('.')
```

Comment:
node has always been a first-class consumer as far as deepmerge is concerned – I'm not okay with moving away from

```js
const deepmerge = require('deepmerge')
```

for node.js/CommonJS